### DISCUSSION TITLE
How to contact a translator?

### DISCUSSION THREAD

**Discussion by tttp** [WeblateOrg/weblate#6081] [Q&A] [answered]
Hi,

One of the joy of weblate is when someone we don't know yet contributes a new language or improves an existing one. However, sometimes, their contributions aren't easy to use, for instance we had recently someone that improved a lot of strings, but unfortunately that translated the variables name (so {{country}} became {{land}}). Another example was a serbian translator replaced half of the text from cyrilic to latin (I have no knowledge about what it should be, but I'm pretty sure half latin half cyrilic isn't correct ;)

I might be missing something obvious, but is there a way to send a message to the translators? In this example I'd like to explain to them that the variables shouldn't be translated or trying to understand what's the best charset to use. but it might as well be to thank them for their contribution ;)

**Comment by nijel** [comment, marked as answer]
There is currently no such feature in Weblate (see https://github.com/WeblateOrg/weblate/issues/5533). You can get the translator e-mail from the Git commits, though.

**Comment by FutureLearnSole** [comment]
> ... , but for private repositories or for push URLs the setup is more complex and requires authentication.

https://docs.weblate.org/en/latest/vcs.html#git-remote-helpers

https://docs.weblate.org/en/latest/vcs.html#git-remote-helpers
